cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/ScheduleStoreTableUtil.java

```diff
@@ -48,6 +48,6 @@ public String getMetaTableName() {
    * @param datasetFramework framework to add types and datasets to
    */
   public static void setupDatasets(DatasetFramework datasetFramework) throws IOException, DatasetManagementException {
-    datasetFramework.addInstance(OrderedTable.class.getName(), SCHEDULE_STORE_DATASET_NAME, DatasetProperties.EMPTY);
+    datasetFramework.addInstance(Table.class.getName(), SCHEDULE_STORE_DATASET_NAME, DatasetProperties.EMPTY);
```

Comment:
Yes, was thinking about this as well.  I'll file a JIRA for upgrade handling and will be testing the process.